kratos/python/add_mesh_to_python.cpp

```diff
@@ -70,6 +60,11 @@ typedef MeshType::NodesContainerType NodesContainerType;
 typedef Geometry<Node<3> >::PointsArrayType NodesArrayType;
 typedef Geometry<Node<3> >::IntegrationPointsArrayType IntegrationPointsArrayType;
 
+array_1d<double,3> GetNormalFromCondition( Condition& dummy )
```

Comment:
We were agree to define the LocalPointWhenProjected projected, the thing is that I needed the normal, so I defined the method, but it is true that for high order geometries is not defined, I will modify that